recipes/parsplice/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "parsplice" %}
+{% set version = "1.1.1" %}
+{% set build = 0 %}
+{% set build_string = "py{}_{}_{}".format(environ.get('CONDA_PY', ''), mpi, build) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://gitlab.com/exaalt/parsplice/-/archive/master/parsplice-master.tar.gz
```

Comment:
This is for master not 1.1.1

Reply:
The issue is that 1.1 fails to build: 

```
/home/conda/staged-recipes/build_artifacts/parsplice_1580027643965/_build_env/bin/../lib/gcc/x86_64-conda_cos6-linux-gnu/7.3.0/../../../../x86_64-conda_cos6-linux-gnu/bin/ld: CMakeFiles/dbextract.dir/dbextract/main.cpp.o: in function `ParSpliceSplicer::~ParSpliceSplicer()':
main.cpp:(.text._ZN16ParSpliceSplicerD2Ev[_ZN16ParSpliceSplicerD5Ev]+0x1af): undefined reference to `boost::log::v2_mt_posix::attribute_set::~attribute_set()'
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/dbextract.dir/build.make:220: dbextract] Error 1
make[1]: *** [CMakeFiles/Makefile2:82: CMakeFiles/dbextract.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 50%] Linking CXX executable parsplice-md
/home/conda/staged-recipes/build_artifacts/parsplice_1580027643965/_build_env/bin/../lib/gcc/x86_64-conda_cos6-linux-gnu/7.3.0/../../../../x86_64-conda_cos6-linux-gnu/bin/ld: CMakeFiles/parsplice-md.dir/md/main.cpp.o: in function `ParSpliceSplicer::~ParSpliceSplicer()':
main.cpp:(.text._ZN16ParSpliceSplicerD2Ev[_ZN16ParSpliceSplicerD5Ev]+0x1af): undefined reference to `boost::log::v2_mt_posix::attribute_set::~attribute_set()'
collect2: error: ld returned 1 exit status
make[2]: *** [CMakeFiles/parsplice-md.dir/build.make:220: parsplice-md] Error 1
make[1]: *** [CMakeFiles/Makefile2:109: CMakeFiles/parsplice-md.dir/all] Error 2
make: *** [Makefile:130: all] Error 2
```

while the current master builds just fine, so I wait for the developers to release a successor of 1.1 but till then I would be happy to have the current master version in conda-forge.

Reply:
maybe you can use the exact commit? https://gitlab.com/exaalt/parsplice/-/archive/4a48ef86be45746b769f6ba02a3da71125bc7b75/parsplice-4a48ef86be45746b769f6ba02a3da71125bc7b75.tar.gz

Reply:
Done